src/main/java/seedu/duke/manager/DishStorageManager.java

```diff
@@ -0,0 +1,38 @@
+package seedu.duke.manager;
+
+import seedu.duke.entities.Dish;
+
+import java.util.ArrayList;
+
+public class DishStorageManager extends Manager {
+    private static final String FILE = "dish";
```

Comment:
> I think the FILE string needs to have the extension as well (e.g. dish.txt)

It's stored in binary so I'm not sure how to name it. I can change it later.

Reply:
Oh y not txt file though

Reply:
I think it's .bin

But for bin file is it we can just pass in the object to store like that and it will work? I nvr rly work with bin file that often so I'm not sure haha

Reply:
> I think it's .bin
> 
> But for bin file is it we can just pass in the object to store like that and it will work? I nvr rly work with bin file that often so I'm not sure haha

make sense, maybe we decide later?

Reply:
Yeah we can decide that later bah I'll just approve ur PR first everything else looks good